src/bebone/core/types/utils/trait_bryan_angles.h

```diff
@@ -0,0 +1,158 @@
+#ifndef _BEBONE_CORE_TYPES_TRAIT_BRYAN_ANGLES_
+#define _BEBONE_CORE_TYPES_TRAIT_BRYAN_ANGLES_
+
+#include "../matrix/matrix_4x4.tpp"
+
+namespace bebone::core {
+    static inline Matrix<f32, 4, 4> trait_bryan_angle_xzy(const Vec3f& rotation) {
+        const f32 c3 = std::cos(rotation.z);
+        const f32 s3 = std::sin(rotation.z);
+        const f32 c2 = std::cos(rotation.x);
+        const f32 s2 = std::sin(rotation.x);
+        const f32 c1 = std::cos(rotation.y);
+        const f32 s1 = std::sin(rotation.y);
+
+        #ifndef BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER
```

Comment:
Grammar error
Change `BEBONE_TYPES_MATRIX_COLLUM_MAJOR_ORDER` to `BEBONE_TYPES_MATRIX_COLUMN_MAJOR_ORDER`